src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/dashboard/dashboard.component.ts

```diff
@@ -24,33 +26,50 @@ import { SummaryService } from '~/app/shared/services/summary.service';
 export class DashboardComponent implements OnInit, OnDestroy {
   detailsCardData: DashboardDetails = {};
   osdSettings$: Observable<any>;
-  interval = new Subscription();
   permissions: Permissions;
   enabledFeature$: FeatureTogglesMap$;
   color: string;
   capacity$: Observable<any>;
+  usedCapacity$: Observable<any>;
+  iops$: Observable<any>;
+  latency$: Observable<any>;
+  clientThroughput$: Observable<any>;
+  usedCapacitySubs: Subscription;
+  iopsSubs: Subscription;
+  latencySubs: Subscription;
+  clientThroughputSubs: Subscription;
+  readonly promqls = [
+    'sum(ceph_osd_stat_bytes_used)/sum(ceph_osd_stat_bytes)',
+    'sum(ceph_osd_op_rw)',
+    'avg(ceph_osd_commit_latency)',
+    'sum(ceph_osd_recovery_bytes)'
+  ];
+  readonly source = timer(0, 10000);
```

Comment:
Let's change this name to something more meaningful and we will need to change the value on the fly.